tests/unit/amo/utils/test_index.js

```diff
@@ -67,7 +67,19 @@ describe(__filename, () => {
     });
 
     describe('isInternal prop', () => {
-      it('returns true for a URL containing the current host', () => {
+      it('returns true for a single slash-prefixed URL', () => {
+        expect(checkInternalURL({ urlString: pathname }).isInternal).toEqual(
+          true,
+        );
+      });
+
+      it('returns false for a double slash-prefixed URL', () => {
+        expect(
+          checkInternalURL({ urlString: '//path/name' }).isInternal,
```

Comment:
Something like this would be more accurate:

```suggestion
          checkInternalURL({ urlString: '//example.com/name' }).isInternal,
```

Reply:
This is becoming irrelevant with the removal of code for the double-slash URLs, but thanks!